neptune/new/exceptions.py

```diff
@@ -775,6 +779,36 @@ def __init__(self):
         super().__init__(message.format(**STYLES))
 
 
+class NeptuneOfflineModeChangeStageException(NeptuneOfflineModeException):
+    def __init__(self):
+        message = """
+        TODO: NPT-11349
+{h1}
+----NeptuneOfflineModeChangeStageException---------------------------------------------------
+{end}
+It seems you are trying to change stage, while working in an offline mode.
+You need to work in non-offline connection mode to change stage.
+
+You can set connection mode when creating a new run:
+    {python}run = neptune.init(mode="async"){end}
```

Comment:
example shouldn't list a run, as it's a model_version feature

Reply:
Let's don't change this for now. I've added `TODO: NPT-11349` since it's going to be udpated in the same story.

Reply:
Yeah, but you're actually adding incorrect docstring in this PR, this is error-prone IMO. Either add a correct one or do as in `NeptuneProtectedPathException` (empty message).

Reply:
Right, ok.